openshift/rh-che.app.yaml

```diff
@@ -347,6 +347,6 @@ objects:
       name: rhche-host
 parameters:
 - name: IMAGE
-  value: registry.devshift.net/che/rh-che-server
+  value: quay.io/openshiftio/che-rh-che-server
```

Comment:
@jmelis https://github.com/openshiftio/saas-openshiftio/pull/956/files

It looks like the image in saas-openshiftio is different 
- `quay.io/openshiftio/rhel-che-rh-che-server`
- `quay.io/openshiftio/che-rh-che-server`

Reply:
sounds good to me, but I just realised it should be preffixed with `rhel-` to indicate that it's rhel based. making the change, thanks for reviewing!!